internal/tfschema/tfschema.go

```diff
@@ -84,7 +86,33 @@ func detectProviderName(resource hclwrite.Block) (string, error) {
 	return extractProviderNameFromResourceType(terraform.GetResourceType(resource))
 }
 
-func getClient(providerName string, dir string) (tfschema.Client, error) {
+func getTerragruntPluginPath(dir string) string {
```

Comment:
It's non trivial to locate the "plugin/client" path to .terraform (since the folder is a bunch of randomly generated strings). Therefore, had to locate it.

E.g.:  ./.terragrunt-cache/yHtqnMrVQOISIYxobafVvZbAAyU/ThyYwttwki6d6AS3aD5OwoyqIWA/.terraform